packages/tsx-editor/src/components/EditorWrapper.tsx

```diff
@@ -1,22 +1,16 @@
 import * as React from 'react';
-import { Spinner, SpinnerSize } from 'office-ui-fabric-react/lib/Spinner';
-import { Stack } from 'office-ui-fabric-react/lib/Stack';
-import { getWindow, isIE11, getId } from 'office-ui-fabric-react/lib/Utilities';
-import { getMonacoConfig } from '@uifabric/monaco-editor/lib/configureEnvironment';
-import { transformExample } from '../transpiler/exampleTransform';
-import { getSetting } from '../utilities/settings';
+import { getId } from 'office-ui-fabric-react/lib/Utilities';
 import { EditorPreview } from './EditorPreview';
-import { ITextModel, IEditorProps } from './Editor.types';
 import { IEditorWrapperProps, IEditorPreviewProps } from './EditorWrapper.types';
 import { EditorError } from './EditorError';
 import { TypeScriptSnippet } from './TypeScriptSnippet';
+import { EditorLoading } from './EditorLoading';
+import { SUPPORTED_PACKAGES } from '../utilities/defaultSupportedPackages';
+import { isEditorSupported } from '../utilities/isEditorSupported';
 
-// This file MUST NOT directly load the main Editor module which depends on Monaco.
-// It can reference the module for types, but not use it at runtime.
-import * as editorModuleType from './Editor';
-
-// Override setTimeout from Node.js which is not actually the one used here
-declare function setTimeout(callback: Function, delay: number): number;
+// This file MUST NOT directly load the main TsxEditor module which depends on Monaco, to avoid
+// pulling it into a bundle. Importing/rendering with React.lazy solves this.
+const TsxEditorLazy = React.lazy(() => import(/* webpackChunkName: 'tsx-editor-core' */ './TsxEditor'));
```

Comment:
Any reason why we need to specifically name the chunk other than aesthetics (or caching?) This disables certain built-in size-based optimizations that webpack does with their chunking, I believe

Reply:
Hmm okay. I can un-name it if you think that's better.